group_vars/gxconfig.yml

```diff
@@ -2243,6 +2284,18 @@ base_app_main: &BASE_APP_MAIN
   # handler processes. Float values are allowed.
   #workflow_monitor_sleep: 1.0
 
+  # In which cases Galaxy should calculate a hash for a new dataset.
+  # Dataset hashes can be used by the Galaxy job cache/search to check
+  # if job inputs match. Setting the 'enable_celery_tasks' option to
+  # true is also required for dataset hash calculation. Possible values
+  # are: 'always', 'upload' (the default), 'never'. If set to 'upload',
+  # the hash is calculated only for the outputs of upload jobs.
+  #calculate_dataset_hash: upload
+
+  # Hash function to use if 'calculate_dataset_hash' is enabled.
+  # Possible values are: 'md5', 'sha1', 'sha256', 'sha512'
+  #hash_function: sha256
```

Comment:
I will just trust the default.

Reply:
This is okay, but we can't change this in the future (unless it's addressed somehow; I haven't reviewed the codebase, so I'm leaning a bit towards `sha512`).